numpy/f2py/rules.py

```diff
@@ -1296,11 +1296,11 @@ def buildmodule(m, um):
                 '%% This file is auto-generated with f2py (version:%s)\n' % (f2py_version))
             if 'shortlatex' not in options:
                 f.write(
-                    '\\documentclass{article}\n\\usepackage{a4wide}\n\\begin{document}\n\\tableofcontents\n\n')
+                    '\\documentclass{article}\n\\usepackage{a4wide}\n\\begin{document}\n\	ableofcontents\n\n')
```

Comment:
blind replacement doesn't always work :).

I wonder if these shouldn't be 4 spaces rather than the tab character though, we don't really use hard-tabs anywhere.

Reply:
Switched to 4 spaces, and yup, I had forgotten about the `tex` commands.. fixed ^_^